test/e2e/apps/rc.go

```diff
@@ -41,6 +46,14 @@ import (
 var _ = SIGDescribe("ReplicationController", func() {
 	f := framework.NewDefaultFramework("replication-controller")
 
+	var ns string
+	var dc dynamic.Interface
+
+	ginkgo.BeforeEach(func() {
+		ns = f.Namespace.Name
+		dc = f.DynamicClient
```

Comment:
nit: ditto: Use `f.DynamicClient` directly.

Reply:
I implemented it this way because it was written like this in [test/e2e/apps/disruption.go:L65-L72](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/disruption.go#L65-L72)